src/data_processing/wiki_to_netflix.py

```diff
@@ -0,0 +1,136 @@
+import requests
+import csv
+
+base_dir = os.path.join(os.path.dirname(__file__), '../data/')
+user_agent = 'Noisebridge MovieBot 0.0.1/Audiodude <audiodude@gmail.com>'
+
+missing_count = 0
+num_rows = 42
+
+# Reading netflix text file
+def read_netflix_txt(txt_file):
+    with open(txt_file, "r", encoding = "ISO-8859-1") as netflix_data:
+        netflix_list = [line.rstrip().split(',', 2) for line in netflix_data.readlines()[0:num_rows]]
```

Comment:
Just be aware here that you're reading all 17000 rows, but throwing away all but `num_rows`. It might be better to do a for loop `for i in range(num_rows):` and then build the list that way.

Reply:
Probably not an actual issue because it's only 17000 rows. In "production" we are reading all the rows anyway. Resolving.